Sources/JSON.swift

```diff
@@ -295,6 +295,42 @@ extension Dictionary {
     }
 }
 
+// MARK: - Raw
+
+extension JSON: RawRepresentable {
+    
+    enum JSONDataError: ErrorType {
+        case ObjectMissing
+        case InvalidJSONObject
+    }
+    
+    public init?(rawValue: AnyObject) {
+        guard NSJSONSerialization.isValidJSONObject(rawValue) else {
+            self.init()
```

Comment:
No need for the `self.init()` here; we can simply `return nil`, if rawValue is not a valid JSON object